front/pages/w/[wId]/welcome.tsx

```diff
@@ -149,11 +149,14 @@ export default function Welcome({
               Let's check a few things.
             </p>
           </div>
-          <div>
-            <p>
-              Your email is <span className="font-bold">{user.email}</span>.
-            </p>
-          </div>
+          {!isAdmin && (
+            <div>
+              <p>
+                Your will be joining the workspace of{" "}
```

Comment:
nit: "You will be joining the workspace: X" seems maybe a bit more resilient as wording?